Jenkinsfile

```diff
@@ -1,14 +1,22 @@
-#!/usr/bin/env groovy
 @Library('jenkins_lib')_
 
 pipeline {
-  agent any
+  agent {label 'nebula-slave03'}
 
```

Comment:
why it is fixed agent,it should be any

Reply:
i have made changes as per master branch